src/server/req_rescq.c

```diff
@@ -611,18 +644,104 @@ req_confirmresv(struct batch_request *preq)
 		rc = reply_text(presv->ri_brp, PBSE_NONE, buf);
 		presv->ri_brp = (struct batch_request *)0;
 	}
+
 	svr_mailownerResv(presv, MAIL_CONFIRM, MAIL_NORMAL, log_buffer);
-	if (presv->ri_wattr[RESV_ATR_interactive].at_flags & ATR_VFLAG_SET) {
-		(void)sprintf(buf, "requestor=%s@%s Interactive=%ld",
+	if (presv->ri_wattr[RESV_ATR_interactive].at_flags & ATR_VFLAG_SET)
+		presv->ri_wattr[RESV_ATR_interactive].at_flags &= ~ATR_VFLAG_SET;
+
+	if (presv->ri_wattr[RESV_ATR_resv_standing].at_val.at_long) {
+		(void)sprintf(buf, "requestor=%s@%s start=%ld end=%ld nodes=%s count=%ld",
 			preq->rq_user, preq->rq_host,
-			presv->ri_wattr[RESV_ATR_interactive].at_val.at_long);
+			presv->ri_qs.ri_stime, presv->ri_qs.ri_etime,
+			next_execvnode,
+			presv->ri_wattr[RESV_ATR_resv_count].at_val.at_long);
 	} else {
-		(void)sprintf(buf, "requestor=%s@%s", preq->rq_user, preq->rq_host);
+		(void)sprintf(buf, "requestor=%s@%s start=%ld end=%ld nodes=%s",
+			preq->rq_user, preq->rq_host,
+			presv->ri_qs.ri_stime, presv->ri_qs.ri_etime,
+			next_execvnode);
 	}
 	if (!is_degraded)
 		account_recordResv(PBS_ACCT_CR, presv, buf);
 
+	free(next_execvnode);
+
+	if (is_being_altered) {
+		/*
+		 * If the reservation's start time is being altered after the reservation had started
+		 * running, we need to stop the reservation queue as it would been started at the
+		 * original start time. This will prevent any jobs - that are submitted after the
+		 * reservation's start time is changed - from running.
+		 * The reservation went to CO from RN while being altered, that means the reservation
+		 * had resources assigned. We should decrement their usages until it starts running
+		 * again, where the resources will be accounted again.
```

Comment:
this is not a race condition. Maybe I can make the comment more clear. 
If the reservation was running and its start time was changed, there is a possibility that the reservation will start late, so we have to take back the resources.
And we stop the queue so that the jobs in it don't start running.